<html>
<head>
  <title>VEP summary</title>
  <script type="text/javascript" src="http://www.google.com/jsapi"></script>
  <script type="text/javascript">
    google.load('visualization', '1', {packages: ['corechart','table']});
  </script>
  <script type="text/javascript">
    
    function init() {
      // charts
      var var_class_pie = drawPie('var_class_pie', 'Variant classes', google.visualization.arrayToDataTable([['Variant class','Count'],['insertion',78],['deletion',116],['SNV',2918]]), null);
var var_class_table = drawTable('var_class_table', 'Variant classes', google.visualization.arrayToDataTable([['Variant class','Count'],['insertion',78],['deletion',116],['SNV',2918]]));

          google.visualization.events.addListener(var_class_pie, 'select', function() {
            var_class_table.setSelection(var_class_pie.getSelection());
          });
          google.visualization.events.addListener(var_class_table, 'select', function() {
            var_class_pie.setSelection(var_class_table.getSelection());
          });
        var var_cons_pie = drawPie('var_cons_pie', 'Consequences (most severe)', google.visualization.arrayToDataTable([['Consequence type','Count'],['splice_acceptor_variant',1],['start_lost',1],['inframe_insertion',2],['inframe_deletion',1],['missense_variant',30],['splice_donor_5th_base_variant',2],['splice_region_variant',9],['splice_donor_region_variant',4],['splice_polypyrimidine_tract_variant',7],['synonymous_variant',46],['5_prime_UTR_variant',4],['3_prime_UTR_variant',7],['non_coding_transcript_exon_variant',12],['intron_variant',47],['downstream_gene_variant',1],['intergenic_variant',2938]]), {slices: [{color: "#ff7f50"}, {}, {color: "#ff69b4"}, {color: "#ff69b4"}, {color: "#ffd700"}, {}, {color: "#ff7f50"}, {}, {}, {color: "#76ee00"}, {}, {}, {}, {color: "#02599c"}, {color: "#a2b5cd"}, {color: "gray"}]});
var var_cons_table = drawTable('var_cons_table', 'Consequences (most severe)', google.visualization.arrayToDataTable([['Consequence type','Count'],['splice_acceptor_variant',1],['start_lost',1],['inframe_insertion',2],['inframe_deletion',1],['missense_variant',30],['splice_donor_5th_base_variant',2],['splice_region_variant',9],['splice_donor_region_variant',4],['splice_polypyrimidine_tract_variant',7],['synonymous_variant',46],['5_prime_UTR_variant',4],['3_prime_UTR_variant',7],['non_coding_transcript_exon_variant',12],['intron_variant',47],['downstream_gene_variant',1],['intergenic_variant',2938]]));

          google.visualization.events.addListener(var_cons_pie, 'select', function() {
            var_cons_table.setSelection(var_cons_pie.getSelection());
          });
          google.visualization.events.addListener(var_cons_table, 'select', function() {
            var_cons_pie.setSelection(var_cons_table.getSelection());
          });
        var consequences_pie = drawPie('consequences_pie', 'Consequences (all)', google.visualization.arrayToDataTable([['Consequence type','Count'],['splice_acceptor_variant',1],['start_lost',2],['inframe_insertion',11],['inframe_deletion',3],['missense_variant',112],['splice_donor_5th_base_variant',3],['splice_region_variant',73],['splice_donor_region_variant',10],['splice_polypyrimidine_tract_variant',41],['synonymous_variant',140],['5_prime_UTR_variant',18],['3_prime_UTR_variant',30],['non_coding_transcript_exon_variant',124],['intron_variant',450],['NMD_transcript_variant',87],['non_coding_transcript_variant',86],['upstream_gene_variant',291],['downstream_gene_variant',257],['intergenic_variant',2938]]), {slices: [{color: "#ff7f50"}, {}, {color: "#ff69b4"}, {color: "#ff69b4"}, {color: "#ffd700"}, {}, {color: "#ff7f50"}, {}, {}, {color: "#76ee00"}, {}, {}, {}, {color: "#02599c"}, {}, {}, {color: "#a2b5cd"}, {color: "#a2b5cd"}, {color: "gray"}]});
var consequences_table = drawTable('consequences_table', 'Consequences (all)', google.visualization.arrayToDataTable([['Consequence type','Count'],['splice_acceptor_variant',1],['start_lost',2],['inframe_insertion',11],['inframe_deletion',3],['missense_variant',112],['splice_donor_5th_base_variant',3],['splice_region_variant',73],['splice_donor_region_variant',10],['splice_polypyrimidine_tract_variant',41],['synonymous_variant',140],['5_prime_UTR_variant',18],['3_prime_UTR_variant',30],['non_coding_transcript_exon_variant',124],['intron_variant',450],['NMD_transcript_variant',87],['non_coding_transcript_variant',86],['upstream_gene_variant',291],['downstream_gene_variant',257],['intergenic_variant',2938]]));

          google.visualization.events.addListener(consequences_pie, 'select', function() {
            consequences_table.setSelection(consequences_pie.getSelection());
          });
          google.visualization.events.addListener(consequences_table, 'select', function() {
            consequences_pie.setSelection(consequences_table.getSelection());
          });
        var coding_pie = drawPie('coding_pie', 'Coding consequences', google.visualization.arrayToDataTable([['Consequence type','Count'],['start_lost',2],['inframe_insertion',11],['inframe_deletion',3],['missense_variant',112],['synonymous_variant',140]]), {slices: [{}, {color: "#ff69b4"}, {color: "#ff69b4"}, {color: "#ffd700"}, {color: "#76ee00"}]});
var coding_table = drawTable('coding_table', 'Coding consequences', google.visualization.arrayToDataTable([['Consequence type','Count'],['start_lost',2],['inframe_insertion',11],['inframe_deletion',3],['missense_variant',112],['synonymous_variant',140]]));

          google.visualization.events.addListener(coding_pie, 'select', function() {
            coding_table.setSelection(coding_pie.getSelection());
          });
          google.visualization.events.addListener(coding_table, 'select', function() {
            coding_pie.setSelection(coding_table.getSelection());
          });
        var chr_bar = drawBar('chr_bar', 'Variants by chromosome', google.visualization.arrayToDataTable([['Chromosome','Count'],['1',3112]]), {legend: {position: "none"}});
var chr_table = drawTable('chr_table', 'Variants by chromosome', google.visualization.arrayToDataTable([['Chromosome','Count'],['1',3112]]));

          google.visualization.events.addListener(chr_bar, 'select', function() {
            chr_table.setSelection(chr_bar.getSelection());
          });
          google.visualization.events.addListener(chr_table, 'select', function() {
            chr_bar.setSelection(chr_table.getSelection());
          });
        var chr_1_area = drawArea('chr_1_area', 'Distribution of variants on chromosome 1', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',27],['1',40],['2',26],['3',31],['4',1],['5',5],['6',29],['7',17],['8',7],['9',16],['10',10],['11',19],['12',32],['13',2],['14',3],['15',9],['16',58],['17',53],['18',11],['19',33],['20',27],['21',27],['22',67],['23',19],['24',38],['25',8],['26',43],['27',15],['28',11],['29',15],['30',0],['31',16],['32',33],['33',14],['34',9],['35',11],['36',23],['37',0],['38',32],['39',12],['40',21],['41',13],['42',10],['43',44],['44',27],['45',11],['46',23],['47',17],['48',7],['49',2],['50',3],['51',6],['52',7],['53',17],['54',13],['55',29],['56',0],['57',7],['58',1],['59',15],['60',4],['61',0],['62',23],['63',7],['64',9],['65',21],['66',4],['67',17],['68',4],['69',0],['70',1],['71',1],['72',1],['73',0],['74',3],['75',10],['76',15],['77',4],['78',9],['79',5],['80',0],['81',0],['82',3],['83',0],['84',0],['85',22],['86',36],['87',5],['88',0],['89',19],['90',3],['91',13],['92',14],['93',18],['94',26],['95',5],['96',0],['97',4],['98',2],['99',3],['100',13],['101',4],['102',0],['103',12],['104',0],['105',0],['106',0],['107',1],['108',5],['109',61],['110',28],['111',32],['112',17],['113',10],['114',18],['115',6],['116',10],['117',17],['118',3],['119',2],['120',11],['121',0],['122',0],['123',0],['124',0],['125',0],['126',0],['127',0],['128',0],['129',0],['130',0],['131',0],['132',0],['133',0],['134',0],['135',0],['136',0],['137',0],['138',0],['139',0],['140',0],['141',0],['142',0],['143',0],['144',40],['145',19],['146',8],['147',10],['148',0],['149',1],['150',16],['151',30],['152',51],['153',19],['154',29],['155',26],['156',62],['157',22],['158',49],['159',21],['160',25],['161',32],['162',8],['163',2],['164',0],['165',15],['166',7],['167',16],['168',7],['169',18],['170',5],['171',26],['172',8],['173',2],['174',1],['175',26],['176',12],['177',13],['178',4],['179',15],['180',12],['181',7],['182',9],['183',12],['184',7],['185',1],['186',11],['187',0],['188',0],['189',0],['190',0],['191',0],['192',3],['193',2],['194',0],['195',0],['196',7],['197',16],['198',1],['199',0],['200',14],['201',44],['202',23],['203',16],['204',40],['205',33],['206',17],['207',28],['208',10],['209',20],['210',6],['211',7],['212',17],['213',3],['214',13],['215',7],['216',7],['217',3],['218',1],['219',2],['220',13],['221',6],['222',15],['223',14],['224',8],['225',3],['226',19],['227',16],['228',14],['229',9],['230',12],['231',12],['232',10],['233',3],['234',12],['235',19],['236',49],['237',17],['238',1],['239',0],['240',7],['241',7],['242',6],['243',4],['244',1],['245',8],['246',7],['247',42],['248',50],['249',1]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var protein_bar = drawBar('protein_bar', 'Position in protein', google.visualization.arrayToDataTable([['Position in protein (percentile)','Count'],['00-10%',31],['10-20%',33],['20-30%',18],['30-40%',19],['40-50%',21],['50-60%',32],['60-70%',27],['70-80%',36],['80-90%',36],['90-100%',15]]), {hAxis: {title: "Position in protein (percentile)", textStyle: {fontSize: 10}}, legend: {position: "none"}});

    }
    
    function drawPie(id, title, data, options) {    
      var pie = new google.visualization.PieChart(document.getElementById(id));
      pie.draw(data, options);
      return pie;
    }
    function drawBar(id, title, data, options) {
      var bar = new google.visualization.ColumnChart(document.getElementById(id));
      bar.draw(data, options);
      return bar;
    }
    function drawTable(id, title, data) {
      var table = new google.visualization.Table(document.getElementById(id));
      table.draw(data, null);
      return table;
    }
    function drawLine(id, title, data, options) {
      var line = new google.visualization.LineChart(document.getElementById(id));
      line.draw(data, options);
      return line;
    }
    function drawArea(id, title, data, options) {
      var area = new google.visualization.AreaChart(document.getElementById(id));
      area.draw(data, options);
      return area;
    }
    google.setOnLoadCallback(init);
  </script>
  
  
  <style type="text/css">
    body {
      font-family: arial, sans-serif;
      margin: 0px;
      padding: 0px;
    }
    
    a {color: #36b;}
    a.visited {color: #006;}
    
    .stats_table {
      margin: 5px;
      table-layout: fixed;
      width: 100%;
    }

    pre {
      white-space: normal;
      word-break: keep-all;
    }

    td {
      padding: 5px;
      word-break: break-word;
    }

    td:nth-child(2) {
      width: 80%;
    }

    th.gradient {
      height: auto;
    }

    .stats_table tr:nth-child(odd) {
      background-color: #f0f0f0;
    }
    
    h3 {
      color: #666;
    }
    
    .masthead {
      background-color: white;
      color: rgb(204, 221, 255);
      height: 80px;
      width: 100%;
      padding: 0px;
    }
    
    .main {
      padding: 10px;
    }
    
    .gradient {
      background: #333366; /* Old browsers */
      background: -moz-linear-gradient(left,  #333366 0%, #ffffff 100%); /* FF3.6+ */
      background: -webkit-gradient(linear, left top, right top, color-stop(0%,#333366), color-stop(100%,#ffffff)); /* Chrome,Safari4+ */
      background: -webkit-linear-gradient(left,  #333366 0%,#ffffff 100%); /* Chrome10+,Safari5.1+ */
      background: -o-linear-gradient(left,  #333366 0%,#ffffff 100%); /* Opera 11.10+ */
      background: -ms-linear-gradient(left,  #333366 0%,#ffffff 100%); /* IE10+ */
      background: linear-gradient(to right,  #333366 0%,#ffffff 100%); /* W3C */
      filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#333366', endColorstr='#ffffff',GradientType=1 ); /* IE6-9 */
      
      padding: 0px;
      height: 80px;
      width: 700px;
    }
    
    .main_content {
      margin-left: 300px;
    }
    
    .sidemenu {
      width: 260px;
      position: fixed;
      border-style: solid;
      border-width: 2px;
      border-color: rgb(51, 51, 102);
    }
    
    .sidemenu_head {
      width: 250px;
      background-color: rgb(51, 51, 102);
      color: rgb(204, 221, 255);
      padding: 5px;
    }
    
    .sidemenu_body {
      width: 250px;
      padding: 5px;
    }
  </style>
</head>
<body>
<div id="masthead" class="masthead">
  <div style="float: left; display: inline; padding: 10px; height: 80px;">
    <a href="http://www.ensembl.org/"><img src="http://static.ensembl.org/i/e-ensembl.png" width="185" height="45" alt="Ensembl"></a>
  </div>
  
  <div style="float: right; display: inline; height: 80px; background: white; padding: 10px;">
    <a href="http://www.ensembl.org/vep"><img src="http://www.ensembl.org/img/vep_logo.png" width="111" height="50" alt="VEP"></a>
  </div>
  <div class="gradient">
  </div>
</div>
<div class="main">
<div class="sidemenu"><div class="sidemenu_head">Links</div><div class="sidemenu_body"><ul><li><a href="#masthead">Top of page</a></li><li><a href="#run_stats">VEP run statistics</a></li><li><a href="#data_version">Data version</a></li><li><a href="#gen_stats">General statistics</a></li><li><a href="#var_class">Variant classes</a></li><li><a href="#var_cons">Consequences (most severe)</a></li><li><a href="#consequences">Consequences (all)</a></li><li><a href="#coding">Coding consequences</a></li><li><a href="#chr">Variants by chromosome</a></li><li><a href="#protein">Position in protein</a></li></ul></div></div><div class='main_content'><h3 id="run_stats">VEP run statistics</h3><table class="stats_table"><tr><td>VEP version (API)</td><td> 115.2 (115)</td></tr><tr><td>Annotation sources</td><td>Cache: <kbd>/mnt/d/clovertex_assignment/homo_sapiens/115_GRCh37</kbd></td></tr><tr><td>Species</td><td>homo_sapiens</td></tr><tr><td>Command line options</td><td><pre>--cache  --dir_cache /mnt/d/clovertex_assignment/ --fasta /home/ritika/vep_minidata/GRCh37.primary.fa --force_overwrite --input_file /mnt/d/clovertex_assignment/test1_data 1.vcf --offline --output_file /mnt/d/clovertex_assignment/test1_data_annotated.vcf --vcf</pre></td></tr><tr><td>Start time</td><td>2025-10-01 01:23:44</td></tr><tr><td>End time</td><td>2025-10-01 01:24:01</td></tr><tr><td>Run time</td><td>17 seconds</td></tr><tr><td>Input file</td><td><pre>/mnt/d/clovertex_assignment/test1_data 1.vcf</pre></td></tr><tr><td>Output file</td><td><pre>/mnt/d/clovertex_assignment/test1_data_annotated.vcf</pre></td></tr></table><h3 id="data_version">Data version</h3>No data version information available.<h3 id="gen_stats">General statistics</h3><table class="stats_table"><tr><td>Lines of input read</td><td>30726</td></tr><tr><td>Variants processed</td><td>3112</td></tr><tr><td>Variants filtered out</td><td>0</td></tr><tr><td>Novel / existing variants</td><td>-</td></tr><tr><td>Overlapped genes</td><td>77</td></tr><tr><td>Overlapped transcripts</td><td>494</td></tr><tr><td>Overlapped regulatory features</td><td>-</td></tr></table><hr/><h3 id="var_class">Variant classes</h3><div id="var_class_pie" style="width: 800px; height: 200px">&nbsp;</div><div id="var_class_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="var_cons">Consequences (most severe)</h3><div id="var_cons_pie" style="width: 800px; height: 400px">&nbsp;</div><div id="var_cons_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="consequences">Consequences (all)</h3><div id="consequences_pie" style="width: 800px; height: 400px">&nbsp;</div><div id="consequences_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="coding">Coding consequences</h3><div id="coding_pie" style="width: 800px; height: 400px">&nbsp;</div><div id="coding_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr">Variants by chromosome</h3><div id="chr_bar" style="width: 800px; height: 200px">&nbsp;</div><div id="chr_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr_1">Distribution of variants on chromosome 1</h3><div id="chr_1_area" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="protein">Position in protein</h3><div id="protein_bar" style="width: 800px; height: 200px">&nbsp;</div></div>
</div></body>
</html>
